install-bindata.sh

```diff
@@ -0,0 +1,6 @@
+go get -u github.com/go-bindata/go-bindata/... 
+cd ${GOPATH}/src/github.com/go-bindata/go-bindata/
+go test
+cd go-bindata
+go build 
+mv go-bindata /usr/local/bin/
```

Comment:
does it not automatically build it for you?. also if you have your GOBIN set then the mv shouldn't be necessary